public/app/features/dashboard/dashgrid/PanelHeader/PanelHeader.tsx

```diff
@@ -59,12 +61,21 @@ export const PanelHeader: FC<Props> = ({ panel, error, isViewing, isEditing, dat
                   />
                 ) : null}
                 <h2 className={styles.titleText}>{title}</h2>
-                <Icon name="angle-down" className="panel-menu-toggle" />
-                <PanelHeaderMenuWrapper panel={panel} dashboard={dashboard} show={panelMenuOpen} onClose={closeMenu} />
-                {data.request && data.request.timeInfo && (
-                  <span className="panel-time-info">
-                    <Icon name="clock-nine" size="sm" /> {data.request.timeInfo}
-                  </span>
+                {!isPublicDashboardView() && (
+                  <div data-testid="panel-dropdown">
+                    <Icon name="angle-down" className="panel-menu-toggle" />
+                    <PanelHeaderMenuWrapper
+                      panel={panel}
+                      dashboard={dashboard}
+                      show={panelMenuOpen}
+                      onClose={closeMenu}
+                    />
+                    {data.request && data.request.timeInfo && (
+                      <span className="panel-time-info">
+                        <Icon name="clock-nine" size="sm" /> {data.request.timeInfo}
+                      </span>
```

Comment:
this is important info that is part of the panel header

Reply:
okay I'll keep that. What is `data.request.timeInfo` supposed to represent? It's blank on my local instance

Reply:
if the panel has a time shift or time range offset it will show that